mozdownload/scraper.py

```diff
@@ -265,6 +311,7 @@ def total_seconds(td):
 
         self.logger.info('Downloading from: %s' % self.url)
         self.logger.info('Saving as: %s' % self.filename)
+        self.logger.info('Directory: %s' % os.path.split(self.url)[0])
```

Comment:
Do you mean that it's better to add an url parameter to existing download method so we can download both the checksum and the real file with the same method?
I don't download the checksum to a file anymore, I store it in a variable for fast access. This way I think that a separate download method for checksum is simpler.